include/gsl/span

```diff
@@ -629,69 +637,69 @@ private:
     // in subspans and constructors from arrays
     constexpr span(KnownNotNull ptr, index_type count) : storage_(ptr, count) {}
 
-    template <std::ptrdiff_t CallerExtent>
+    template <std::size_t CallerExtent>
     class subspan_selector
     {
     };
 
-    template <std::ptrdiff_t CallerExtent>
+    template <std::size_t CallerExtent>
     span<element_type, dynamic_extent> make_subspan(index_type offset, index_type count,
                                                     subspan_selector<CallerExtent>) const
     {
         const span<element_type, dynamic_extent> tmp(*this);
         return tmp.subspan(offset, count);
     }
 
-    GSL_SUPPRESS(bounds.1) // NO-FORMAT: attribute
+    GSL_SUPPRESS(bounds .1) // NO-FORMAT: attribute
     span<element_type, dynamic_extent> make_subspan(index_type offset, index_type count,
                                                     subspan_selector<dynamic_extent>) const
     {
-        Expects(offset >= 0 && size() - offset >= 0);
```

Comment:
Another no-op >= 0. Probably best to audit for these.

Reply:
good catch, I thought I removed this before I pushed.